includes/class-wc-payments-woopay-direct-checkout.php

```diff
@@ -71,6 +73,12 @@ public function scripts() {
 			return;
 		}
 
+		$woopay_util = new WooPay_Utilities();
```

Comment:
We usually pass other classes objects in the constructor. This allows us to easily mock dependencies in tests. Is there a reason to not do that here?

Reply:
Fixed on https://github.com/Automattic/woocommerce-payments/pull/9792/commits/08b14133dd96229a8763174d2256505d521dbc93.